### DISCUSSION TITLE
Grouped Rows

### DISCUSSION THREAD

**Discussion by amaidah** [nick-keller/react-datasheet-grid#127] [Q&A] [answered]
Hi Nick, is there a way currently to group rows in the grid around a selected column?

Right now, I'm using two separate tables but this has some issues around sorting and UX:
![image](https://user-images.githubusercontent.com/16659483/153254622-7acd9343-d3ad-45cb-8e82-b44b17205024.png)

I'd like to be able to select a column, in this case `current` or `planned` and have the rows group around those selections something like this:
![image](https://user-images.githubusercontent.com/16659483/153255162-17b2b674-1db0-4bc3-abb0-f7b42d46d4de.png)

What do you think?

**Comment by nick-keller** [comment]
Good news: it is definitely possible.
Bad news: it requires a bit of code.

Here is a related thread: https://github.com/Equify/react-datasheet-grid/discussions/58

I might add it to the doc as an example !

**Comment by nick-keller** [comment, marked as answer]
Example available in the doc: https://react-datasheet-grid.netlify.app/docs/examples/collapsible-rows
